src/FileUpload/FileUploadFactory.php

```diff
@@ -53,6 +63,9 @@ public function create($upload, $server)
         $fileupload = new FileUpload($upload, $server);
         $fileupload->setPathResolver($this->pathresolver);
         $fileupload->setFileSystem($this->filesystem);
+        if (null !== $this->fileNameGenerator) {
```

Comment:
Nitpick:  Can this be `$this->fileNameGenerator === null` . I believe other parts of the codebase read like that

Reply:
Never mind :)